phplintinstall

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Download PHPCS
+mkdir bin
+cd bin
+curl -OL https://squizlabs.github.io/PHP_CodeSniffer/phpcs.phar
```

Comment:
To make things faster and avoid unnecessary downloads, I'd suggest checking first if the file exists:

```
if [ ! -f phpcs.phar ]; then
  curl -OL https://squizlabs.github.io/PHP_CodeSniffer/phpcs.phar
fi